visallo/visallo
# Add ability to expand/collapse long string properties in element inspector

<img alt="expand and collapse property" src="https://cl.ly/1H1C3v0d0i1I/Screen%20Recording%202017-08-11%20at%2003.12%20PM.gif" height="400px"/>

The expanded state is stored per property and will persist until you refresh your browser.

Also added scroll overflow for justification in the popover and property form and truncation in element inspector headers and vertex connection popover.

Testing Instructions:

1. Add a new string property with a very long value.
1. The property should be truncated and have an option to expand/collapse when hovering.
1. Expand the property.
1. Close the element inspector then reopen.
1. The property should still be expanded.


CHANGELOG
Added: Ability to collapse and expand long text properties in the element inspector.
Fixed: `formatters.vertex.prop` and `formatters.vertex.propDisplay` can also provide an array of arguments to pass to optional transforms. (e.g. `formatters.vertex.propDisplay('myPropertyIRI', 'property value', { truncate: [5] })`
Fixed: When pasting a large amount of text as justification, display it with a scrollbar rather than filling the screen to fit.
